eval/src/vespa/eval/instruction/generic_rename.cpp

```diff
@@ -69,10 +69,13 @@ generic_rename(const Value &a,
                const ValueType &res_type, const ValueBuilderFactory &factory)
 {
     auto cells = a.cells().typify<CT>();
-    std::vector<string_id> output_address(sparse_plan.mapped_dims);
-    std::vector<string_id*> input_address;
+    SmallVector<string_id> output_address;
```

Comment:
there is no need for an explicit loop here, since you can create a small vector with n default constructed objects directly